components/Maps/MapChart.js

```diff
@@ -39,8 +39,8 @@ const MapChart = () => {
                   geography={geo}
                   fill="#DDD"
                   onClick={() => {
-                    const { name } = geo.properties;
-                    window.location.assign(`state/${name}`);
+                    const curState = allStates.find((s) => s.val === geo.id);
```

Comment:
This is not that slow but we probably don't need to do this on every click since all 50 states will have the same id and that most likely won't change. So can we just create a json of the  state objects that has the `s.val` as the key and then the value could be another object that has the `id` and import it. Like:

```json
{
    "newyork": {
        "id": 1,
    },...
}

```